src/Parser/Mage/Fire/Modules/Features/HeatingUp.js

```diff
@@ -0,0 +1,163 @@
+    if (!GUARANTEE_CRIT_SPELLS.includes(spellId) || (spellId === SPELLS.PHOENIXS_FLAMES.id && (this.targetId !== damageTarget))) {
+      return;
+    }
+    const hasHeatingUp = this.combatants.selected.hasBuff(SPELLS.HEATING_UP.id);
+    const hasHotStreak = this.combatants.selected.hasBuff(SPELLS.HOT_STREAK.id);
+
+    switch (spellId) {
+      case SPELLS.FIRE_BLAST.id:
+        if (hasHotStreak) {
+          this.fireBlastWithHotStreak += 1;
+          debug && console.log("Fire Blast with Hot Streak @ " + formatMilliseconds(event.timestamp - this.owner.fight.start_time));
+        }
+        if (!hasHeatingUp && !hasHotStreak) {
+          this.fireBlastWithoutHeatingUp += 1;
+          debug && console.log("Fire Blast without Heating Up @ " + formatMilliseconds(event.timestamp - this.owner.fight.start_time));
+        }
+        break;
+      case SPELLS.PHOENIXS_FLAMES.id:
+        if (hasHotStreak) {
+          this.phoenixFlamesWithHotStreak += 1;
+          debug && console.log("Phoenix Flames with Hot Streak @ " + formatMilliseconds(event.timestamp - this.owner.fight.start_time));
+        }
+        if (!hasHeatingUp && !hasHotStreak) {
+          this.phoenixFlamesWithoutHeatingUp += 1;
+          debug && console.log("Phoenix Flames without Heating Up @ " + formatMilliseconds(event.timestamp - this.owner.fight.start_time));
+        }
+        break;
+      default:
+        return;
+    }
+  }
+
+  get fireBlastWasted() {
+    return this.fireBlastWithoutHeatingUp + this.fireBlastWithHotStreak;
+  }
+
+  get phoenixFlamesWasted() {
+    return this.phoenixFlamesWithoutHeatingUp + this.phoenixFlamesWithHotStreak;
+  }
+
+  get totalWasted() {
+    return this.fireBlastWasted + this.phoenixFlamesWasted;
+  }
+
+  get fireBlastUtil() {
+    return 1 - this.fireBlastMissedPercent;
+  }
+
+  get phoenixFlamesUtil() {
+    return 1 - this.phoenixFlamesMissedPercent;
+  }
+
+  get fireBlastMissedPercent() {
+    return this.fireBlastWasted / this.abilityTracker.getAbility(SPELLS.FIRE_BLAST.id).casts;
+  }
+
+  get phoenixFlamesMissedPercent() {
+    return this.phoenixFlamesWasted / this.abilityTracker.getAbility(SPELLS.PHOENIXS_FLAMES.id).casts;
+  }
+
+  suggestions(when) {
+		when(this.fireBlastMissedPercent).isGreaterThan(0.1)
+			.addSuggestion((suggest, actual, recommended) => {
+				return suggest(<Wrapper>You cast <SpellLink id={SPELLS.FIRE_BLAST.id} /> {this.fireBlastWithHotStreak} times while <SpellLink id={SPELLS.HOT_STREAK.id}/> was active and {this.fireBlastWithoutHeatingUp} times while you didnt have <SpellLink id={SPELLS.HEATING_UP.id}/>. Make sure that you are only using Fire Blast to convert Heating Up into Hot Streak.</Wrapper>)
+					.icon(SPELLS.FIRE_BLAST.icon)
+					.actual(`${formatPercentage(this.fireBlastMissedPercent)}% missed`)
+					.recommended(`<${formatPercentage(recommended)}% is recommended`)
+					.regular(0.1).major(0.2);
+			});
+    when(this.phoenixFlamesMissedPercent).isGreaterThan(0.1)
```

Comment:
You should use the new suggestion thresholds format even if this isn't going in the checklist.